src/swagger-specs/profile.yaml

```diff
@@ -4,22 +4,29 @@ info:
   description: >- 
     Real-time Customer Profile enables you to drive coordinated, consistent and relevant experiences for your customers no matter where or when they interact with your brand. Profile provides a holistic view of each individual customer that combines data from multiple channels, including online, offline, CRM, and third-party data.
```

Comment:
This one could also probably have the user-centric restructuring in the second sentence, like the others